README.md

```diff
@@ -25,6 +25,11 @@ In general, these are the guidelines we consider when deciding what to pre-insta
 - Tools and versions will typically be removed 6 months after they are deprecated or have reached end-of-life.
 - If a tool can be installed during the build, we will evaluate how much time is saved
  and how much space is used by having the tool pre-installed.
+- If a tool license is not MIT, Apache or GNU, we will have to check it with legal.

```

Comment:
MIT, Apache, and GNU licenses are ok, anything else we'll have to check with lawyers 💼

Reply:
Thanks, updated this line